src/index.js

```diff
@@ -76,31 +76,79 @@ function setup() {
 	const nestedSettings = VIP_GOVERNANCE.nestedSettings;
 	const nestedSettingPaths = getNestedSettingPaths( nestedSettings );
 
+	// pull out all the wildcard block names.
+	const nestedWildcardPaths = Object.keys( nestedSettingPaths ).reduce( ( acc, blockName ) => {
```

Comment:
Optimized it based on your suggestion and it def feels a touch faster

Reply:
Thank you! This is what I was picturing. I pushed up a small update to split these in one pass imperatively, a tiny optimization since we're going for speed here:

https://github.com/Automattic/vip-governance-plugin/blob/9b80bbaf4757075bd984d24ca49b21dd7ae6295c/src/index.js#L79-L90